deploy/kubernetes/base/node.yaml

```diff
@@ -76,7 +73,6 @@ spec:
             - name: registration-dir
               mountPath: /registration
         - name: liveness-probe
-          imagePullPolicy: Always
```

Comment:
Same

Reply:
imagePullPolicy was not added in helm (so no need to change).